lib/luhn_cases.rb

```diff
@@ -0,0 +1,27 @@
+require 'exercise_cases'
+
+class LuhnCase < OpenStruct
+  def name
+    'test_%s' % description.tr('- ', '__')
+  end
+
+  def work_load
```

Comment:
Is there a style guide somewhere (other than the README)? Because some of the exercises use `work_load` and others use `workload`. In fact, as long as it is the same in both `example.tt` and `<exercise>_cases.rb` it should work fine.